src/sentry/integrations/slack/utils.py

```diff
@@ -173,6 +180,22 @@ def build_rule_url(rule, group, project):
     return absolute_uri(rule_url)
 
 
+def build_upgrade_notice_attachment(group):
+    org_slug = group.organization.slug
+    url = absolute_uri(
+        u"/settings/{}/integrations/slack/?tab=configurations&referrer=slack".format(org_slug)
```

Comment:
could they be coming from somewhere else in slack to this page? just wonder if `slack` is specific enough, might be totally fine

Reply:
@MeredithAnya We used `referrer=slack` for the link to the issues page so I wanted to be consistent here.